buildpack.toml

```diff
@@ -1,7 +1,7 @@
 api = "0.2"
 
 [buildpack]
-  id = "org.cloudfoundry.bundler"
+  id = "paketo-community.bundler"
```

Comment:
Should this be `paketo-community/builder` instead?
Otherwise these buildpacks won't be publishable to the buildpack registry.

Reply:
Yup, good catch. Thanks!